### DISCUSSION TITLE
How can I make View Collection for Multiple Relation

### DISCUSSION THREAD

**Discussion by codeforfunzz** [pocketbase/pocketbase#4509] [Q&A] [answered]
Hi, I have a multiple relation like this "demand_codes_id":["0zbmzohfqphg5i5","xu036p7m76b4bjm","itw2sf5un4sok3t","b948d54vdax0qpf","ery3oy39cyar5sp"]

![pocketbase1](https://github.com/pocketbase/pocketbase/assets/153375958/ed54afd5-0731-45ac-896a-912a817b17a8)

how can I create a View Collection to select the content instead of id?
for example, I have collection like this
0zbmzohfqphg5i5: BUYING,
xu036p7m76b4bjm: SELLING,
itw2sf5un4sok3t: RENT,
b948d54vdax0qpf: PRIMARY_BUYING,
ery3oy39cyar5sp: SECONDARY_BUYING,

I want to make a View Collection that show "demand_codes_id":["BUYING","SELLING","RENT","PRIMARY_BUYING","SECONDARY_BUYING"]

pls help me, Thanks

**Comment by codeforfunzz** [comment]
@ganigeorgiev pls help me

**Reply by ganigeorgiev** [reply]
_Please don't ping for no reason. I cannot provide dedicated support._

If you need help, provide more details about your collections structure, not just a screenshot because at the moment it is not clear from where "BUYING", "SELLING", etc. values come from.

**Comment by codeforfunzz** [comment]
Sorry, first time use discussions on github, just think it's like Discord or Telegram, mention someone when need help.

About my project, this is my collection

![image](https://github.com/pocketbase/pocketbase/assets/153375958/9219c75b-b328-4c6e-b6ce-7d43a0eb53a4)
![image](https://github.com/pocketbase/pocketbase/assets/153375958/668adf26-5cf3-4877-ad94-035170059859)

I mean I want demand_codes_id column display value 'demand_codes' instead of 'id'

**Reply by ganigeorgiev** [reply, marked as answer]
If you just need it for display purposes in the Admin UI, then you can simply mark the `demand_codes` field as "Presentable" from its field options (_the cogwheel next to the field_).

If you really need a View collection that will always return the relations `demand_codes` field value, one possible query could be:

```sql
-- for all available json SQLite functions see https://www.sqlite.org/json1.html
SELECT
  test_relation.id,
  json_group_array(dc.demand_codes) as codes
FROM test_relation
LEFT JOIN json_each(test_relation.demand_codes_id) je
LEFT JOIN demand_codes dc ON dc.id = je.value
GROUP BY test_relation.id
```
_Note that you'll lose the relation reference and the "codes" column will be treated as `json`._

**Reply by codeforfunzz** [reply]
Thanks for your help, it works like a charm
